charts/mgmt-agent/README.md

```diff
@@ -31,6 +36,18 @@ A Helm chart for collecting Kubernetes Metrics using OCI Management Agent into O
 | oci-onm-common.resourceNamePrefix | string | `"{{ .Values.global.resourceNamePrefix }}"` | Prefix to be attached to resources created through this chart. Not all resources may have this prefix. |
 | oci-onm-common.serviceAccount | string | `"{{ .Values.global.resourceNamePrefix }}"` | Name of the Kubernetes ServiceAccount |
 | serviceAccount | string | `"{{ .Values.global.resourceNamePrefix }}"` | Name of the Kubernetes ServiceAccount |
+| deployment.security.runAsUser | integer | `1000` | Processes in the Container will use the specified user ID |
+| deployment.security.runAsGroup | integer | `2000` | Processes in the Container will use the specified group ID |
+| deployment.security.fsGroup | integer | `2000` | Files created in the Container will use the specified group ID |
+| deployment.cleanupEpochTime | integer | `nil` | Please provide the current epoch time in seconds (date +%s) to clean up the agent installation directory from previous deployment |
```

Comment:
Can you use the same comment that I put in the oci-onm's Readme.md for this?

Reply:
done